.github/workflows/ci.yml

```diff
@@ -28,7 +28,11 @@ jobs:
             -scheme PINOperation \
             ONLY_ACTIVE_ARCH=NO \
             CODE_SIGNING_REQUIRED=NO \
+            CLANG_ANALYZER_OUTPUT=plist-html \
+            CLANG_ANALYZER_OUTPUT_DIR="$(pwd)/clang" \
           | xcpretty
+          if [[ -n `find $(pwd)/clang -name "*.html"` ]] ; then rm -rf $(pwd)/clang; exit 1; fi
+          rm -rf $(pwd)/clang
```

Comment:
I don't think you need to use `$(pwd)` throughout there. You should be able to just use a relative path (e.g. `./clang`).